first section added to 12_0.rmd

wwjxza opened the pull request:
Hi @dapeng, I just added the first section to the bot chapter. Please review it and let me know if any changes are required. Thanks. Have a nice day!

@wangwenjing

wwjxza committed:
first section added to 12_0.rmd

wwjxza commented:
I just previewed the file on GitHub and found that my chapter structure does not match the default, but I cannot edit it. Please solve this issue when you are available. Thanks.

pzhaonet commented:
Thanks for your contribution. As the chapter numbers are compiled automatically, I have removed them from your draft. I have added your name as the section author as well. Please feel free to revise it as you like.

wwjxza commented:
Thanks!

Have a wonderful day!



On Tue, Feb 20, 2018 at 3:22 AM Peng Zhao <notifications@github.com> wrote:

> Thanks for your contribution. As the chapter numbers are compiled
> automatically, I have removed them from your draft. I have added your name
> as the section author as well. Please feel free to revise it as you like.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pzhaonet/steem-guides/pull/15#issuecomment-366932662>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AK1AA2M92XfxFCEXqUSHluKAMIAGQsWlks5tWpz6gaJpZM4SLaf1>
> .
>
-- 

--------------------------------------------
Wenjing Wang
Ph.D.
Department of Mechanical Engineering
University of Alberta
Edmonton, AB, Canada
e-mail: wenjing3@ualberta.ca
--------------------------------------------
